### PR TITLE
feat(20221): Apply maxzoom for Bivariate and MCDA layers based on Indicators maxZoom property

### PR DESCRIPTION
**Pull request by albaranau**
https://kontur.fibery.io/Tasks/Task/FE-Apply-maxzoom-for-bivariate-MCDA-layers-based-on-indicators-metadata-20221

## Summary by CodeRabbit

- **New Features**
	- Enhanced indicator structure with a new `direction` property for better context.
	- Dynamic calculation of maximum zoom levels for MCDA and bivariate layers.

- **Bug Fixes**
	- Removed unused import statements to clean up code.

- **Tests**
	- Introduced a new test suite for validating maximum zoom level calculations.
	- Added test cases to ensure correct functionality of zoom level determination.

- **Documentation**
	- Updated comments in the `MCDALayer` interface regarding future changes.

### CODE REVIEW

File: src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
   FALLBACK_BIVARIATE_MAX_ZOOM,
   FALLBACK_BIVARIATE_MIN_ZOOM,
 } from '../../BivariateRenderer/constants';
+import { getMaxMCDAZoomLevel } from '../../../../../utils/bivariate/getMaxZoomLevel';
```
Comment (vkozio) [inline src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts]: we have aliases, use shorter ~utils/bivariate...

File: src/utils/bivariate/getMaxZoomLevel.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import type { MCDAConfig } from '../../core/logical_layers/renderers/stylesConfigs/mcda/types';
```
Comment (vkozio) [inline src/utils/bivariate/getMaxZoomLevel.ts]: ~core/...

**Review by vkozio — APPROVED** [review]
see comments

File: src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts (hunk 1 above)
Comment (albaranau) [reply src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts]: Fixed

File: src/utils/bivariate/getMaxZoomLevel.ts (hunk 2 above)
Comment (albaranau) [reply src/utils/bivariate/getMaxZoomLevel.ts]: Fixed

**Pull request merged**